includes/settings-page.php

```
@@ -96,279 +101,424 @@ function rt_settings_init() {
+		<?php esc_html_e('Check this if you want to disable all CSS optimizations.', 'RT_Script_Optimizer'); ?>
 	</p>
-	<?php
+<?php
+}
+
+/**
+ * Field callback to disable JS optimizations.
+ *
+ * @param array $args arguments passed.
+ */
+function rt_scripts_optimizer_disable_js_optimizations_callback($args)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$args'. (Unused Code Rules)

(UnusedFormalParameter)